docs/plugins.asciidoc

```diff
@@ -13,24 +13,18 @@ A list of existing Kibana plugins is available on https://github.com/elastic/kib
 Use the following command to install a plugin:
 
 [source,shell]
-bin/kibana plugin --install <org>/<package>/<version>
+bin/kibana-plugin install <package> (optional URL)
 
-You can also use `-i` instead of `--install`, as in the following example:
-
-[source,shell]
-bin/kibana plugin -i elasticsearch/marvel/latest
-
-Because the organization given is `elasticsearch`, the plugin management tool automatically downloads the 
-plugin from `download.elastic.co`.
+When you use `elasticsearch` as the organization name, the plugin management tool downloads the plugin from 
+`download.elastic.co`.
 
 [float]
-=== Installing Plugins from an Arbitrary URL
+==== Installing Plugins from an Arbitrary URL
 
-You can specify a URL to a plugin with the `-u` or `--url` options after the `-i` or `--install` option, as in the
-following example:
+You can specify the URL to a plugin after the plugin name, as in the following example:
```

Comment:
As described above, you now either specify a name OR you specify a url to a zip.  You do not specify both.